.env-example

```diff
@@ -4,3 +4,7 @@ MAPIT_URL="https://mapit.mysociety.org/"
 MAPIT_API_KEY=""
 GOOGLE_ANALYTICS=""
 GOOGLE_SITE_VERIFICATION=""
+
+TEST_AIRTABLE_BASE_ID="appWEAndAXvvkWOOk"
```

Comment:
Should these be committed? Remembering this is a public repository? I think it is okay, as these are meaningless without the API key, which can be a GitHub secret.

Reply:
Yes I think it's fine. Ultimately it's just a test instance.